ogr/services/gitlab.py

```diff
@@ -41,13 +52,16 @@ class GitlabService(GitService):
     def __init__(self, token=None, url=None, full_repo_name=None):
         super().__init__(token=token)
         url = url or "https://gitlab.com"
-        self.g = gitlab.Gitlab(url=url, private_token=token)
+        self.g = gitlab.Gitlab(url=url, private_token=token, ssl_verify=False)
         self.g.auth()
-        self.user = self.g.users.list(username=self.g.user.get_username())[0]
         self.repo = None
         if full_repo_name:
             self.repo = self.g.projects.get(full_repo_name)
 
+    @property
+    def user(self) -> GitUser:
+        return self.g.users.list(username=self.g.user.get_username())[0]
+
```

Comment:
To have the same API as GitHub/PAgure, we need to create a `GitlabProject` class.

- Feel free to remove unused methods form `GitlabService`. (e.g. `create_from_remote_url`, `is_fork_of`, ...) and move project-specific methods to newly created `GitlabProject`.